frameworks/g-cloud-9/questions/services/onlineTIcketingSupportAccessibility.yml

```diff
@@ -0,0 +1,28 @@
+name: "Online ticketing support accessibility"
+question: "Does your online ticketing support comply with the Web Content Accessibility Guidelines 2.0?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: radios
+options:
+    -
+        label: "AAA"
```

Comment:
The labels here don't match the question.  If you ask "does it comply" then the answer should be yes or no, not "AAA" or whatever.

Assuming AAA, etc. are levels of compliance then the question should be more like "To what level does your ticketing support comply...", with the final option being "It doesn't comply at any of these levels"

I realise this pull request probably isn't the place to raise questions about the content, but you could mention to @superroz that there are some things I mentioned here that you might or might not care about my opinion on 😄